docs/reference/testing.md

```
@@ -36,7 +36,36 @@ When `ry test` is run without arguments, it:
 
 ## Syntax
 
-### describe / it
+### Function-Based Syntax (recommended)
+
+Use the `@it` and `@describe` directives to define test cases as ordinary named functions:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)